src/engine/sys/sdl_glimp.cpp

```diff
@@ -1131,60 +1108,31 @@ static void GLimp_InitExtensions()
 		glConfig2.drawBuffersAvailable = true;
 	}
 
-#ifdef GL_ARB_get_program_binary
-	if( GLEW_ARB_get_program_binary )
+	glConfig2.getProgramBinaryAvailable = false;
 	{
 		int formats = 0;
 
 		glGetIntegerv( GL_NUM_PROGRAM_BINARY_FORMATS, &formats );
 
-		if ( !formats )
-		{
-			logger.Notice("...GL_ARB_get_program_binary found, but with no binary formats");
-			glConfig2.getProgramBinaryAvailable = false;
-		}
-		else
+		if ( formats == 0 )
 		{
-			logger.Notice("...using GL_ARB_get_program_binary");
-			glConfig2.getProgramBinaryAvailable = true;
+			logger.WithoutSuppression().Notice("...no program binary formats");
```

Comment:
No need for `WithoutSuppression` for something which can only be printed once per renderer restart.

Reply:
What about other notices? like `...using` or `...ignoring`? We don't use hundreds of extension, maybe we don't need to suppress them at all…